composer.json

```diff
@@ -34,6 +35,9 @@
         "doctrine/mongodb-odm": "~1.0",
         "doctrine/doctrine-bundle": "~1.0",
         "doctrine/orm": "~2.2",
+        "doctrine/couchdb": "1.0.x-dev",	
+        "doctrine/couchdb-odm": "dev-master",
```

Comment:
TBH this is the problem I'm seeing here, I'd like for them to do a release before we integrate it.

Reply:
Totally agree, CouchDB 2.1 was just release so I hope some people can update this...

Reply:
Yeah, nobody is working on couchdb-odm at the moment. I wouldn't count on a stable release anytime soon. With that in mind, I'm not sure if we should add integration for CouchDB at this time.